### ISSUE TITLE
Repomd.xml doesn't get downloaded while using steps mentioned in the documentation

### ISSUE THREAD

**Issue by strikingraghu** [kubernetes/kubernetes#107120]
### What happened?

While installing K8s on CentOS VM, below steps didn't work seamlessly:

Site Link - https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/
----------------------------------------------------------------------------------------------------------------------------
Code Section:

cat <<EOF | sudo tee /etc/yum.repos.d/kubernetes.repo
[kubernetes]
name=Kubernetes
baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-\$basearch
enabled=1
gpgcheck=1
repo_gpgcheck=1
gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
exclude=kubelet kubeadm kubectl
EOF

----------------------------------------------------------------------------------------------------------------------------

I see an issue in the baseurl getting 404 error, which needs to be adjusted/modified accordingly

Solution:
baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-\$basearch [DOESN'T WORK]
baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64 [WORKS WELL]

Can someone validate this and update the documentation?

### What did you expect to happen?

I see an issue in the baseurl getting 404 error, which needs to be adjusted/modified accordingly

### How can we reproduce it (as minimally and precisely as possible)?

Installing kubeadm, kubelet and kubectl via below provided packages link:

----------------------------------------------------------------------------------------------------------------------------
cat <<EOF | sudo tee /etc/yum.repos.d/kubernetes.repo
[kubernetes]
name=Kubernetes
baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-\$basearch
enabled=1
gpgcheck=1
repo_gpgcheck=1
gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
exclude=kubelet kubeadm kubectl
EOF

# Set SELinux in permissive mode (effectively disabling it)
sudo setenforce 0
sudo sed -i 's/^SELINUX=enforcing$/SELINUX=permissive/' /etc/selinux/config
sudo yum install -y kubelet kubeadm kubectl --disableexcludes=kubernetes
sudo systemctl enable --now kubelet
----------------------------------------------------------------------------------------------------------------------------

In the above mentioned code, this line would create a problem.
baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-\$basearch

### Kubernetes version

<details>

```console
$ kubectl version
Latest Version Applicable
```

</details>

### Cloud provider

<details>
On-Prem Installation
</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
CentOS-8
# paste output here
$ uname -a
# paste output here
Linux x86_64
# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

### Container runtime (CRI) and and version (if applicable)

<details>
Containerd.io
But the CRI is not relevant for this problem
</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by strikingraghu** [comment]
@sig/docs
Mapping it to sig/docs team to review this piece of suggestion!

**Comment by pacoxu** [comment]
```
       $basearch
          Refers to the base architecture of the system. For example,
          i686 and i586 machines both have a base architecture of i386,
          and AMD64 and Intel64 machines have a base architecture of
          x86_64.
```
See https://man7.org/linux/man-pages/man5/yum.conf.5.html.

**Comment by strikingraghu** [comment]
> ```
>        $basearch
>           Refers to the base architecture of the system. For example,
>           i686 and i586 machines both have a base architecture of i386,
>           and AMD64 and Intel64 machines have a base architecture of
>           x86_64.
> ```
> 
> See https://man7.org/linux/man-pages/man5/yum.conf.5.html.

I agree, but using the link in document fails! [https://packages.cloud.google.com/yum/repos/kubernetes-el7-\$basearch]
Hence, I am requesting to make it simpler to new users to download and install K8s without much issues.

**Comment by pacoxu** [comment]
BTW this is a "problem" in [website](https://github.com/kubernetes/website/).
/transfer-issue website
